docs/content/guide/component.ngdoc

```diff
@@ -233,9 +233,13 @@ it upwards to the heroList component, which updates the original data.
 </file>
 
 <file name="heroDetail.js">
-  function HeroDetailController($scope, $element, $attrs) {
+  function HeroDetailController($scope) {
```

Comment:
You are right. Please, do remove the unnecessary `$scope` and update the test to not provide one.